api.lua

```diff
@@ -60,6 +80,10 @@ function skins.update_player_skin(player)
 		if minetest.global_exists("sfinv") and sfinv.enabled then
 			sfinv.set_player_inventory_formspec(player)
 		end
+		-- Voxelibre compatibility
+		if minetest.global_exists("mcl_skins") then
```

Comment:
Useless comment(s) if a self-explaining variable is used:
```suggestion
	if skins.use_voxelibre_compat then
```
and in init.lua:
```Lua
skins.use_voxelibre_compat = minetest.global_exists("mcl_skins")
```

That makes it easier to search for MCL2/VoxelLibre compatibility code, and ensures that every branch is checked the same way in the future (`minetest.get_modpath` is roughly equivalent). Furthermore, that variable could be turned of trivially for testing purposes (or future changes).

Reply:
Thanks, that does look better; got it!